cmd/test/get_fps_dur.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/livepeer/lpms/ffmpeg"
+)
+
+func main() {
```

Comment:
Can you change to a proper unit test?

Reply:
Moved to [ffmpeg/ffmpeg_test.go](https://github.com/livepeer/lpms/pull/407/files#diff-4ab68a4de67768f1acb5be217d4ec78bab27b99381025beedf82c11a5fa9c7ae)